runtime/src/main/scala/akka/grpc/scaladsl/GrpcExceptionHandler.scala

```diff
@@ -26,7 +26,8 @@ object GrpcExceptionHandler {
     case _: UnsupportedOperationException ⇒
       Future.successful(GrpcResponseHelpers.status(Status.UNIMPLEMENTED))
     case other ⇒
-      Future.failed(other)
+      other.printStackTrace()
+      Future.successful(GrpcResponseHelpers.status(Status.INTERNAL))
```

Comment:
I think this makes sense.

Reply:
Agreed. In a future version we might want to add further details to the response with `Status.INTERNAL.withDescription(...)`, but that seems outside the scope of this PR.